appinfo/info.xml

```diff
@@ -6,12 +6,12 @@
 	<author>Thomas Müller, Vincent Petry</author>
 	<summary>Define password policies for user and public link passwords</summary>
 	<description>
-The Password Policy extension enables ownCloud administrators to define password requirements like minimum characters, numbers, capital letters and more for all kinds of password endpoints like user account and public link sharing passwords. To add another layer of security, the administrator can enforce expiration dates for public link shares depending on whether a password has been set or not. As a further measure the extension saves a history of hashed user passwords to prevent users from choosing a former password again, enforcing password security even more. Users can also be required to change their password upon first login. To impose regular password changes administrators can set up password expiration policies. 
+The Password Policy extension enables ownCloud administrators to define password requirements like minimum characters, numbers, capital letters and more for all kinds of password endpoints like user account and public link sharing passwords. To add another layer of security, the administrator can enforce expiration dates for public link shares depending on whether a password has been set or not. As a further measure the extension saves a history of hashed user passwords to prevent users from choosing a former password again, enforcing password security even more. Users can also be required to change their password upon first login. To impose regular password changes administrators can set up password expiration policies. For this users can be notified via mail, web interface and the ownCloud Clients when their password is about to expire and when it has expired.
```

Comment:
notified via email instead of mail ?

mail could refer to snail mail ?

Reply:
hehe, ok. changed to email ;)